indexing-service/src/main/java/org/apache/druid/indexing/common/task/Task.java

```diff
@@ -138,6 +140,15 @@ default int getPriority()
    */
   String getDataSource();
 
+  @Nullable
+  default String getInputSourceType() {
```

Comment:
Maybe should add these functions to a different interface, with non-default, and have the tasks that are now implementing properly, implement that interface, so its easier to know which tasks are properly returning this information